src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -21,7 +21,7 @@ public static IEnumerable<object[]> InputData()
     }
 
     [Theory]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Issue https://github.com/dotnet/corefx/issues/18220")]
+    [ActiveIssue("https://github.com/dotnet/corefx/issues/18220", TargetFrameworkMonikers.UapAot)]
```

Comment:
Yes we do. We have the same overloads for TargetFrameworkMonikers as target platform. Where the issue can be a string or a number: https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Attributes/ActiveIssueAttribute.cs#L19